### ISSUE TITLE
Not possible write bytes as a hash value

### ISSUE THREAD

**Issue by mosquito** [coleifer/vedis-python#13]
I try to serialize values via msgpack, but i can't decode it because vedis calls `str` on the objects.

```python
>>> from vedis import Vedis
>>> db.hset('foo', 'bar', b'\0\1\2\3')
>>> db.hget('foo', 'bar')
"b'\\x00\\x01\\x02\\x03'"
>>> type(db.hget('foo', 'bar'))
str
>>> db.hgetall('foo')
{'bar': "b'\\x00\\x01\\x02\\x03'"}
```

**Comment by coleifer** [comment]
I originally wrote this with python 2... probably need to go back and rework the internals to use bytestrings everywhere.

Out of curiosity why are you using vedis? There are a number of other embedded databases that are much better. Like Sophia ([python bindings](https://github.com/coleifer/sophy)) or kyotocabinet for instance.

**Comment by mosquito** [comment]
It's simple and fast. Has no extra requirements.

I'll have to try sophy, but you should fix this issue 😄

**Comment by coleifer** [comment]
I definitely will, there are just some nice alternatives out there.

**Comment by coleifer** [comment]
d60004e79160d88ed75eb9439920c5fed1ba8a84 and released new version 0.7.0
